packages/amplify-category-geo/src/service-walkthroughs/mapWalkthrough.ts

```diff
@@ -84,10 +84,13 @@ export const mapStyleWalkthrough = async (parameters: Partial<MapParameters>): P
     const mapStyleChoices = [
         { name: 'Streets (data provided by Esri)', value: MapStyle.VectorEsriStreets },
         { name: 'Berlin (data provided by HERE)', value: MapStyle.VectorHereBerlin },
+        { name: 'Explore (data provided by HERE)', value: MapStyle.VectorHereExplore },
+        { name: 'ExploreTruck (data provided by HERE)', value: MapStyle.VectorHereExploreTruck },
         { name: 'Topographic (data provided by Esri)', value: MapStyle.VectorEsriTopographic },
         { name: 'Navigation (data provided by Esri)', value: MapStyle.VectorEsriNavigation },
         { name: 'LightGrayCanvas (data provided by Esri)', value: MapStyle.VectorEsriLightGrayCanvas },
-        { name: 'DarkGrayCanvas (data provided by Esri)', value: MapStyle.VectorEsriDarkGrayCanvas }
+        { name: 'DarkGrayCanvas (data provided by Esri)', value: MapStyle.VectorEsriDarkGrayCanvas },
+        { name: 'Imagery (data provided by Esri)', value: MapStyle.RasterEsriImagery },
```

Comment:
@dayjournal Please remove the trailing comma, the rest of the files don't use it

Reply:
This item has been corrected.